### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/encoder.md`

### PR DESCRIPTION
**Pull request by felipebpl**
In this pull request I added the portuguese **traslation** for the `docs/tutorial/encoder.md`. 

I'm waiting for a response from the maintainers. Just let me know if there's anything I can do to fix any possible problem, but I'm sure that my translation is correct. 

I appreciate any feedback. Thanks!

### CODE REVIEW

File: docs/pt/docs/tutorial/encoder.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Codificador Compatível com JSON
+
+Existem alguns casos em que você pode precisar converter um tipo de dados (como um modelo Pydantic) para algo compatível com JSON (como um `dict`, `list`, etc).
```
Comment (Kludex) [inline docs/pt/docs/tutorial/encoder.md:3]: Are we using "voce" in the docs? 🤔

Comment (felipebpl) [reply docs/pt/docs/tutorial/encoder.md:3]: Hi Kludex, I didn't understand your comment. Are you referring to a typo, or page formality?

If it is about the formality, just let me know that I'll fix it. However,
answering your question, others pt translated pages in the docs also uses "você", so I thought it wouldn't be a problem.

Anyway, let me know how you would like it to be. Thanks!

Comment (felipebpl) [reply docs/pt/docs/tutorial/encoder.md:3]: Ok then.

**Comment by guishas** [comment]
I am from Brazil and I think the translation is fine. 👍

**Review by LorhanSohaky — APPROVED** [review]

**Review by fabioueno — APPROVED** [review]
Nice one! 👏

**Comment by tiangolo** [comment]
Awesome, thank you @felipebpl! :cake: 

And thanks for the reviews @Kludex, @fabioueno, @LorhanSohaky :coffee: :bow:

**Pull request merged**